src/DotvvmAcademy.Web/Course/010_concepts/en/10_viewmodel.md

```diff
@@ -6,16 +6,29 @@ CodeTask:
     Correct: CounterViewModel_20.cs
 ---
 
-# The ViewModel
+# Getting Started: The ViewModel
 
-Hello and welcome to DotVVM Academy. Let's build a simple counter! It will allow users to increment and decrement a value in steps they define.
+Hello and welcome to DotVVM Academy. In this lesson, you'll build a simple Counter application that will allow the users to add or subtract a given step to a value of the counter.
 
-__DotVVM__ is a web framework based on the _Model-View-ViewModel_ (MVVM) pattern.
+> Programming in DotVVM requires basic knowledge of [HTML](https://www.sololearn.com/Course/HTML/) and [C#](https://www.sololearn.com/Course/CSharp/). If you haven't worked with these languages so far, spend some time learning the basic concepts to understand what's going on.
 
-__ViewModels__ are just plain old C# classes that contain the page's data and logic. They get sent between the application and its users.
+How DotVVM works?
+
+Every page in DotVVM consists of two files:
+
+* __View__, which is based on HTML syntax and tells DotVVM how the page will look like.
+* __ViewModel__, which is a C# class that represents the state of the page (values in the form fields) and handles user interactions (button clicks and so on).
+
+The state in ViewModel is kept in C# properties:
+
+```
```

Comment:
Syntax highlighting won't work on this snippet, unless you write `csharp` after the ```.